app/index.js

```diff
@@ -528,8 +528,8 @@ module.exports = Generator.extend({
         if (answers.services.indexOf('AppID') !== -1) {
           this._addService('auth', generateServiceName(this.appname, 'AppID'))
         }
-        if (answers.services.indexOf('Watson Conversation') !== -1) {
-          this._addService('conversation', generateServiceName(this.appname, 'WatsonConversation'))
+        if (answers.services.indexOf('Watson Assistant, formerly Conversation') !== -1) {
+          this._addService('conversation', generateServiceName(this.appname, 'WatsonAssistant'))
```

Comment:
Would it make sense to also change the `ServiceType` to `watsonAssistant`?

Reply:
Unfortunately,  when invoked by scaffolder/starter kit flow in the bluemix object, the label will be "conversation" - we need to make the standalone/prompting mode conform to this convention in serviceType.  This is not a part of the rebrand.

Reply:
oh sad times...